tests/Functional/Ticket/DBAL168Test.php

```diff
@@ -10,7 +10,7 @@ class DBAL168Test extends FunctionalTestCase
 {
     public function testDomainsTable(): void
     {
-        if ($this->connection->getDatabasePlatform() instanceof PostgreSQLPlatform) {
+        if (! $this->connection->getDatabasePlatform() instanceof PostgreSQLPlatform) {
```

Comment:
Since this test has been run and passed against all platforms except for Postgres and is supposed to pass on Postgres as well, why don't we just remove this condition?

Reply:
Yeah, it makes sense to remove, done.